src/modals/cluster-upgrade/data.js

```diff
@@ -254,7 +261,7 @@ function formatEventAsLogEntry ({ id, description, time }) {
   return { id, description, time }
 }
 
-export async function trackUpgradeProgress (correlationId, tick) {
+export async function trackUpgradeProgress (clusterId, correlationId, tick) {
```

Comment:
Any reason for adding `clusterId` argument even if not used within the function?

Reply:
Yes I added that param with the intention that the cluster could be queried directly for the progress update instead of parsing the event text.  I just never did that bit of work.  I'll take it out since it is easy to put back in future if needed.